code/obj/item/clothing/armor.dm

```diff
@@ -434,3 +434,30 @@
 		setProperty("rangedprot", 0.7)
 		setProperty("coldprot", 5)
 		setProperty("heatprot", 35)
+
+/obj/item/clothing/suit/armor/hopuniform
+	name = "burecratic jacket"
+	desc = "A jacket with a bands and stashes you can take on and off as you please"
+	icon_state = "hopuniform1"
+	uses_multiple_icon_states = 1
+	item_state = "hopuniform1"
+	body_parts_covered = TORSO
+	bloodoverlayimage = SUITBLOOD_ARMOR
+
+	attack_self(mob/user as mob)
+		user.show_text("You change out the bands and stashes.")
+		if (src.icon_state == "hopuniform1")
+			src.icon_state = "hopuniform2"
+		else if (src.icon_state == "hopuniform2")
+			src.icon_state = "hopuniform3"
+		else if (src.icon_state == "hopuniform3")
+			src.icon_state = "hopuniform4"
+		else
+			src.icon_state = "hopuniform1"
+
+	setupProperties()
```

Comment:
Too good. HoP is not the captain.

Reply:
I mean, all it is doing is changing out the band and sash, its not like it couldn't be done. do you mean from a gameplay perspective? because they aren't all that different

Reply:
actually is it two sash's? im not sure what technically makes something a sash

Reply:
The properties, not the appearance changing. Seems to be a clone off the cap's?

Reply:
changed it now, its more in line with the other non cap head coats